simulator/src/sequential/SRflipFlop.js

```
@@ -63,7 +63,9 @@ export default class SRflipFlop extends CircuitElement {
      * set this.state to value S.
      */
     resolve() {
-        if (this.reset.value == 1) {
+        if (this.preset.value == 1) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)